app/livechat/server/lib/Livechat.js

```diff
@@ -282,6 +285,9 @@ export const Livechat = {
 			Object.assign(updateUser, { $unset: { department: 1 } });
```

Comment:
We can't remove the department from the visitor object anymore. If the department is not provided, nothing will happend.